app/models/application_search_csv.rb

```diff
@@ -0,0 +1,38 @@
+require 'csv'
+
+class ApplicationSearchCsv
+  def initialize(filter:)
+    @filter = filter
+  end
+
+  def csv
+    @csv ||= CSV.generate do |csv|
+      csv << headers
+
+      search.each do |result|
+        csv << ApplicationSearchResult.new(result).as_csv
```

Comment:
Only thing I was thinking as this ApplicationSearchResult method could be `as_csv_row`? Not a big issue

Reply:
Yeah I thought that then thought CSV stands for "comma separated values" which this is. Perhaps better to rename csv csvs.